### PR TITLE
Mhamzas patch 1

### PR DESCRIPTION
**Pull request by mhamzas**

### CODE REVIEW

File: src/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml [hunk 1]
```diff
@@ -3,4 +3,10 @@
     <context>All</context>
     <endpointUrl>https://chart.googleapis.com</endpointUrl>
     <isActive>true</isActive>
+    <isApplicableToConnectSrc>true</isApplicableToConnectSrc>
```
Comment (pozil) [inline src/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml:6]: This is the only directive that needs to be set to true

File: src/main/default/lwc/gameApp/gameApp.html [hunk 2]
```diff
@@ -112,11 +112,11 @@ <h1 class="answer-label">{correctAnswerLabel}</h1>
             </template>
         </div>
     </div>
-    <div class="slds-text-align_center slds-text-color_weak">
+    <!--<div class="slds-text-align_center slds-text-color_weak">
```
Comment (pozil) [inline src/main/default/lwc/gameApp/gameApp.html:115]: Please revert that change (we want to keep the footer) and change the URL to `https://github.com/fostive/quiz-host-app`

**Review by pozil — CHANGES_REQUESTED** [review]
Thanks the contribution @mhamzas. Could you address the comments I left so that I merge it?

File: src/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml (hunk 1 above)
Comment (mhamzas) [reply src/main/default/cspTrustedSites/googleapis.cspTrustedSite-meta.xml:6]: Right, a single directive will work as well.

File: src/main/default/lwc/gameApp/gameApp.html (hunk 2 above)
Comment (mhamzas) [reply src/main/default/lwc/gameApp/gameApp.html:115]: I am sorry, didn't want to push it.

**Comment by mhamzas** [comment]
Sent another (Updated) pull request. @pozil and thanks for your amazing work!

**Pull request closed without merging**